lib/ReactViews/Analytics/BooleanParameterEditor.jsx

```diff
@@ -11,31 +12,47 @@ const BooleanParameterEditor = React.createClass({
     },
 
     onClick() {
-        const value = this.props.previewed.parameterValues[this.props.parameter.id];
-        this.props.previewed.setParameterValue(this.props.parameter.id, !value);
+        this.props.parameter.value = !this.props.parameter.value;
+    },
+
+    renderCheckbox() {
+        const value = this.props.parameter.value;
+        const name = this.props.parameter.name;
+        const description = this.props.parameter.description;
+
+        return (
+            <div>
+                <button type='button'
+                        className={Styles.btnRadio}
+                        title={description}
+                        onClick={this.onClick}>
+                    {value && <Icon glyph={Icon.GLYPHS.checkboxOn}/>}
+                    {!value && <Icon glyph={Icon.GLYPHS.checkboxOff}/>}
+                    {name}
+                </button>
+            </div>
+        );
     },
 
     renderRadio(state) {
         let name;
         let description;
-        let classNames;
-        const value = this.props.previewed.parameterValues[this.props.parameter.id];
+        const value = this.props.parameter.value === state;
```

Comment:
For booleans that use a radio button (rather than a checkbox), `renderRadio` is called twice, once with `state` true and once with it false.  So I think it will work correctly if the value is false.  Am I missing something?